src/Snowcap/Emarsys/Response.php

```diff
@@ -34,13 +34,13 @@ class Response
      */
     function __construct(array $result = array())
     {
-        if (!isset($result['replyCode']) || !isset($result['replyText']) || !isset($result['data'])) {
+        if (!isset($result['replyCode']) || !isset($result['replyText']) || !array_key_exists('data', $result)) {
```

Comment:
I'm not sure why @mwille did this, but I guess this can be dropped (the effect should be the same).